docs/use_cases.md

```diff
@@ -0,0 +1,74 @@
+# Node.js Use Cases
+
+This doc is intended to capture the key uses cases for Nodej.js. These
+use cases will then be used to guide what we develop/run as part of
+the community benchmarking effort.
+
+# Main Use Case Categories
+
+The folloing categories have been identifed:
+
+* REST or REST like back end services
+* Generating/serving dynamic web page content
+* AJAX like applications with bidirectional communication over
+  websockets and/or http2
+* APIs/microservices based applications
+* Mobile back ends
+* Small scripts
+
+For these use cases one or more of the following are often imporant:
+
+* consistent low latency
+* ability to support high concurrency
+* throughput
+* fast startup/restart/shutdown
+* low resource usage (memory/cpu)
+
+## REST or REST like back end services
+
+Applications using REST
+(https://en.wikipedia.org/wiki/Representational_state_transfer)
+provide access to services over HTTP with verbs such as GET, POST,
+DELETE where URIs are used to identify specific resources.
+Key elements are accepting requests for a given URI and serving
+responses, often using JSON to exchange data.
+
+## Generating/serving dynamic web page content
+
+These applications are often built with additional modules like
+[express](http://expressjs.com), [hapi](http://hapijs.com)
+and [React](http://facebook.github.io/react/).  Key elements are
+building and returning dynamic web pages often using template engines.
+
+## AJAX like applications withh bidirectional communication over websockets and/or http2
```

Comment:
I would remove the reference to AJAX as bidirectional communications implies not using the former.